manager/manager.go

```diff
@@ -375,6 +380,9 @@ type Manager struct {
 	// No thread safety is added around this field, and it MUST NOT be used for any implementation logic.
 	// It should not be used full-stop :).
 	doNotUse_CallOnEachIssue func()
```

Comment:
Can/should this field be removed now, or is this a follow-up? Ref. https://github.com/cert-manager/csi-lib/blob/f3a2df0cf6a71e04bd9a5467fb89dab2e58e5a1a/manager/manager.go#L382-L385

Reply:
This should be a follow-up question in a different issue / PR.